Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+
+## Rbenv & Ruby part
+
+#Install all the requirements
+RUN apt-get install -y curl git build-essential \
+    software-properties-common
+RUN apt-get install -y zlib1g-dev libssl1.0-dev libreadline-dev \
+    libyaml-dev \
+    libffi-dev
+# For newer rubies: libssl-dev libcurl4-openssl-dev
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
+    curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+RUN apt-get update && apt-get install -y nodejs yarn
```

Comment:
I'd great to run them with a single `RUN` command to creating multiple intermediate images and the disk space that takes.

Reply:
I couldn't see these intermediate images while building the app, but I could see that `RUN` is helping stepping the image construction. I gave still reduce the number of calls to 3.